src/main/webapp/data-visualization.js

```diff
@@ -296,3 +299,47 @@ function setStyle(isCountyQuery) {
     amChartsDiv.style.display = 'block';
   }
 }
+
+/**
+ * Create and return an HTML table using the data in dataArray.
+ * Uses first row in dataArray as table headers.
+ */
+function createDataTable(dataArray) {
```

Comment:
I sort of wonder if there's a shorter way to do this, but I can't quite think of any... you could make a createTableRow function but not sure if it really adds any efficiency.

Reply:
Moved the data table creation to inside the create data array function, so we don't loop through the data twice. Let me know your thoughts on this change

Reply:
I think that's a great idea! I see there are some other comments on that section now, but overall the move makes sense to me.